www/.vuepress/config.js

```diff
@@ -15,6 +15,34 @@ const customObjectsComponents = components.filter((comp) =>
   comp.startsWith('components/crayons-custom-objects/')
 );
 
+/** list all versions along with their SHA digest */
+let versionsArr = []; const versionMap = {'v2Components': [], 'v3Components': [], 'v4Components': []}
+
+try {
+  versionsArr = JSON.parse(fs.readFileSync(process.env.VERSION_FILE));
```

Comment:
can you also add a default value if process.env.VERSION_FILE is not set. will be useful when it is run from local

Reply:
added